docs/reference/communication-adapter-interface.md

```diff
@@ -19,9 +19,9 @@ stream driver model for backwards compatibility.
 
 | Kind     | Suggested Name | Port Type             | Usage                                                          |
 |----------|----------------|-----------------------|----------------------------------------------------------------|
-| `input`  | `comDataIn`    | `Drv.ByteStreamSend`  | Port receiving `Fw::Buffer` objects for outgoing transmission. |
+| `input`  | `comDataIn`    | `Svc.ComDataWithContext`  | Port receiving `Fw::Buffer` objects for outgoing transmission. |
 | `output` | `comDataOut`   | `Drv.ByteStreamRecv`  | Port producing incoming `Fw::Buffer` objects.                  |
-| `output` | `comStatus`    | `Fw.SuccessCondition` | Port indicating status of outgoing transmission. See protocol. |
+| `output` | `comStatusOut`    | `Fw.SuccessCondition` | Port indicating status of outgoing transmission. See protocol. |
 
```

Comment:
Need to to document the new return (from driver) and return to (to framer) ports.

Reply:
Fix in this PR.

Reply:
> new return (from driver) 
This port is specific to ComStub which uses a driver under the hood. Omitted from the interface so I think we should omit it from the docs as well ?